spec/spec.md

```diff
@@ -1323,6 +1327,129 @@ The `PermissionQuery` method exists to facilitate lookup of any retained Permiss
     - `delegatedFrom`
     - all properties of `scope` objects
 
+### Subscriptions
+
+Subscriptions to a Decentralized Web Node (DWN) enable efficient data delivery
+between specific DWN clients. This feature is especially valuable in scenarios
+where you have a highly available node (the **source** node) and an ephemeral
+node (the **sink** node) and require the capability to relay real-time updates
```

Comment:
`sink` comes across as a typo for `sync`. If `sink` is intentional, probably best to say so on its first occurrence in any given document.